CHANGELOG.md

```diff
@@ -30,6 +30,7 @@ This release closes the [2.2.0 milestone](https://github.com/Instagram/IGListKit
 - Fix bug where emptyView's hidden status is not updated after the number of items is changed with `insertInSectionController:atIndexes:` or related methods. [Peter Edmonston](https://github.com/edmonston) [(#395)](https://github.com/Instagram/IGListKit/pull/395)
 - Fix bug where `IGListStackedSectionController`'s children need to know `numberOrItems` before didUpdate is called. [(#348)](https://github.com/Instagram/IGListKit/pull/390)
 - Fix bug where `-[UICollectionViewCell ig_setStackedSectionControllerIndex:]` should use `OBJC_ASSOCIATION_COPY_NONATOMIC` for NSNumber. [PhilCai](https://github.com/PhilCai1993) [(#424)](https://github.com/Instagram/IGListKit/pull/426)
+- Rename property `sectionControllerToObjectMap`  to `objectToSectionControllerMap` in `IGSectionMap`. [PhilCai](https://github.com/PhilCai1993) [(#433)](https://github.com/Instagram/IGListKit/pull/436)
```

Comment:
@rnystrom My two cents would not to have entries for structural changes unless it fixes a bug